### DISCUSSION TITLE
Cleanup of resolved discussions

### DISCUSSION THREAD

**Discussion by Hans5958** [ScratchAddons/ScratchAddons#6174] [Discussion]
Currently I'm doing a cleanup where I close resolved and/or duplicate discussion posts. **There is a chance that your notifications will inform you of these closures for every single one of them.** I apologize for such inconvenience, and you may disregard such notifications.

## What is being closed?
- Discussions that have been marked as answered by the original poster
- Discussions where OP has said thanks and request no follow-up
- Discussions that have been indicated as a duplicate of an existing issue/discussion
- Discussions that have been marked as answered and is quite trivial enough
- ~~Past changelogs~~ Canceled due to lower visibility

**Comment by WorldLanguages** [comment]
Ironically, this is a discussion, so I almost miss it because it looks like every other notification 😂

**Comment by BroJac5246** [comment]
I had over 150 notifications today and I was really confused until I realized what was happening 😂

Thanks, though. We really need to go through them!

**Comment by Fluffy728** [comment]
Cool! 

Also, I agree.
